.travis.yml

```diff
@@ -19,7 +19,7 @@ addons:
       - docker-ce
 
 env:
-  - DOCKER_COMPOSE_VERSION=1.25.5
+  - DOCKER_COMPOSE_VERSION=1.25.5 DRIVE_VERSION=0.14.0-dev DAPI_VERSION=0.14.0-dev
```

Comment:
We might just forget about these hardcoded versions. Don't we need just update mn-bootstrap?